packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/ThresholdsFormItem.jsx

```diff
@@ -198,7 +211,7 @@ const ThresholdsFormItem = ({
                   direction="bottom"
                   translateWithId={translateWithId}
                   label=""
-                  items={['>', '<', '=']} // current valid comparison operators
+                  items={comparisonItems} // current valid comparison operators
                   selectedItem={threshold.comparison || '>'}
```

Comment:
if no comparsion may be keep it as =
```suggestion
                  selectedItem={threshold.comparison || '='}
```